.github/workflows/golangci-test.yml

```diff
@@ -0,0 +1,24 @@
+name: golangci-test
```

Comment:
`tests and coverage` might be a better name for this workflow, golangci-lint is the name of the project which runs other linters, also consider changing the filename to something similar